core/src/main/scala/codesearch/core/search/Search.scala

```diff
@@ -49,12 +49,13 @@ trait Search {
             .toList
         } yield CSearchPage(results.sortBy(_.pack.name), entity.lists.size, ErrorResponse(""))
       }
-      case scala.util.Failure(exception) =>
-         {
-           val message = exception.getMessage
-           IO(CSearchPage(Seq.empty[Search.PackageResult], 0,
-             ErrorResponse(message.substring(0,1).toUpperCase + message.substring(1, message.size))))
-         }
+      case scala.util.Failure(exception) => {
```

Comment:
Please don't use Try monad so. You can use cats error hendling via `handleErrorWith` function from cats.effect.IO.